src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -26,8 +42,16 @@ export const StatusBadge: ChakraComponent<
 > = chakra(
   forwardRef<HTMLDivElement, React.PropsWithChildren<StatusBadgeProps>>(
     (props, ref?) => {
-      const { children, className, id, level = "low", ...rest } = props;
-      const styles = useStyleConfig("StatusBadge", { variant: level });
+      const { children, className, id, level, type, ...rest } = props;
+
+      // let finalVariant;
+      // if (!level && !type) finalVariant = "low";
+      // else if (level) finalVariant = level;
+      // else if (type) finalVariant = type;
+
+      const finalVariant = level ? level : type ? type : "low";
```

Comment:
This should also separately include a check to see if `level` is being used to log a deprecation message.

Reply:
While we say the prop is deprecated, like in other components, I did not add a console warning to announce that something is deprecated.  We just never know if/when something will get updated in our consuming apps.